src/porepy/fracs/fracture_network.py

```diff
@@ -4,21 +4,19 @@
 from __future__ import annotations
 
 import warnings
-from typing import Optional, Union
+from typing import List, Optional, Union
 
 import porepy as pp
 from porepy.fracs.fracture_network_2d import FractureNetwork2d
 from porepy.fracs.fracture_network_3d import FractureNetwork3d
-from porepy.fracs.line_fracture import LineFracture
-from porepy.fracs.plane_fracture import PlaneFracture
 
 # Custom typings
-Fracture = Union[LineFracture, PlaneFracture]
+FractureList = Union[List[pp.LineFracture], List[pp.PlaneFracture]]
```

Comment:
```suggestion
FractureList = Union[list[pp.LineFracture], List[pp.PlaneFracture]]
```

Reply:
I tried that first. For some reason, Python 3.8 refused, so I had to revert.